project/MiMaFilters.scala

```diff
@@ -45,5 +45,9 @@ object MiMaFilters {
     ProblemFilters.exclude[InaccessibleClassProblem]("scala.quoted.FromExpr$PrimitiveFromExpr"),
     ProblemFilters.exclude[InaccessibleClassProblem]("scala.quoted.Type$ValueOf$"),
     ProblemFilters.exclude[InaccessibleClassProblem]("scala.reflect.Selectable$DefaultSelectable"),
+
+    // Runtime-only classes that we don't check (user code is not supposed to explicitly use scala.runtime anyway)
```

Comment:
In general we still need to worry about binary compat for scala.runtime since it appears in bytecode, the difference here is that we don't guarantee bincompat when -coverage-out is used, so that should be mentioned explicitly in the comment.